libraries/rush-lib/src/logic/Autoinstaller.ts

```diff
@@ -233,4 +247,100 @@ export class Autoinstaller {
       console.log(message);
     }
   }
+
+  public static getAutoinstallerNames(rushConfiguration: RushConfiguration): string[] {
+    const autoinstallerNames: string[] = [];
+    const files: FolderItem[] = FileSystem.exists(rushConfiguration.commonAutoinstallersFolder)
+      ? FileSystem.readFolderItems(rushConfiguration.commonAutoinstallersFolder)
+      : [];
+
+    for (const file of files) {
+      if (
+        file.isDirectory() &&
+        FileSystem.exists(path.join(rushConfiguration.commonAutoinstallersFolder, file.name, 'package.json'))
+      ) {
+        autoinstallerNames.push(file.name);
+      }
+    }
+
+    return autoinstallerNames;
+  }
+
+  public static async prepareAutoinstallersAsync(
+    rushConfiguration: RushConfiguration,
+    specificNames?: string[]
+  ): Promise<void> {
+    const autoinstallerNames: string[] = [];
+
+    if (specificNames) {
+      autoinstallerNames.push(...specificNames);
+    } else {
+      autoinstallerNames.push(...Autoinstaller.getAutoinstallerNames(rushConfiguration));
+    }
+
+    for (const autoinstallerName of autoinstallerNames) {
+      const autoinstaller: Autoinstaller = new Autoinstaller({
+        autoinstallerName,
+        rushConfiguration
+      });
+      await autoinstaller.prepareAsync();
+      console.log('');
+    }
+
+    console.log(colors.gray('='.repeat(79)));
+
+    if (autoinstallerNames.length === 0) {
+      console.log('\nNo autoinstallers to prepare.');
+      return;
+    }
+
+    console.log(colors.green(`\n${autoinstallerNames.length} autoinstaller(s) prepared.`));
```

Comment:
What does "prepare" mean in this context? I'm not sure a user is going to understand that.